{{cookiecutter.project_slug}}/config/settings/base.py

```diff
@@ -96,21 +94,19 @@
     },
 ]
 
-
 # Internationalization
 # https://docs.djangoproject.com/en/3.1/topics/i18n/
 
-LANGUAGE_CODE = 'en-us'
+LANGUAGE_CODE = 'es-es'
 
-TIME_ZONE = 'UTC'
+TIME_ZONE = 'Cuba'
```

Comment:
This cookiecutter is intended to be international, we must use UTC timezone, not Cuba.